framework/functions/modernize-resources.md

```diff
@@ -0,0 +1,77 @@
+---
+
+layout: default
+tag: modernize-resources
+
```

Comment:
Now that this page exists, we need to link it to the /framework/functions.md

There's an unorganized list of functions with links. We haven't automated this yet, so it's just a manual addition.